steps/deploygate--upload-app-bitrise-step/1.0.1/step.yml

```diff
@@ -0,0 +1,110 @@
+title: DeployGate Upload
+summary: |
+  Distribute your in-development iOS/Android apps to your developers or testers
+description: |
+  This is a step to upload your iOS/Android application to DeployGate. DeployGate enable to share in-development apps with others instantly. For more details, please read references in [DeployGate.com](https://deploygate.com/?locale=en)
+website: https://github.com/DeployGate/upload-app-bitrise-step
+source_code_url: https://github.com/DeployGate/upload-app-bitrise-step
+support_url: https://github.com/DeployGate/upload-app-bitrise-step/issues
+published_at: 2018-09-26T17:29:05.7800137+09:00
+source:
+  git: https://github.com/DeployGate/upload-app-bitrise-step.git
+  commit: b08f5f49e83842ba33425d5e6237a5bdc1b52c07
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: curl
+  apt_get:
+  - name: git
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- api_key: ""
+  opts:
+    description: The authorization token to be used when uploading app
+    is_required: true
+    summary: User's API Key or Organization's API Key
+    title: 'DeployGate: API Key'
+- opts:
+    description: |
+      The owner name of this application
+    is_required: true
+    summary: User name or Organization name
+    title: 'DeployGate: Owner Name'
+  owner_name: ""
+- app_path: ""
+  opts:
+    description: |
+      App's binary file (IPA/APK) to be uploaded.
+      $BITRISE\_APK\_PATH, $BITRISE\_IPA\_PATH, and so on.
```

Comment:
are the slash characters (`\`) a typo in `$BITRISE\_APK\_PATH` and `$BITRISE\_IPA\_PATH`?

Reply:
It is markdown escape.
If this is not used it will be converted to em tag.
<img width="617" alt="2018-09-27 15 08 23" src="https://user-images.githubusercontent.com/851025/46126246-79760d80-c267-11e8-94fe-89fd9d226f6a.png">